kevinpapst/adminlte-bundle/0.9/config/packages/admin_lte.yaml

```diff
@@ -0,0 +1,45 @@
+#
+# Configuration file for the AdminLTE bundle
+#
+# For more information about the bundle settings visit:
+# https://github.com/kevinpapst/AdminLTEBundle/blob/master/Resources/docs/configurations.md
+#
+admin_lte:
+    options:
```

Comment:
Is this all the default config? 

There is no need to add that in a recipe. The bundle will work equally well without it.

Reply:
Hi @Nyholm - thanks for your review! 
Yes its the default config and it isn't necessarily needed. Personally I believe a full default config helps in setting up a module, but I can remove it if it isn't accepted otherwise. 

For sure every user will adapt at least some of the values in here. But I am fine with adapting it if this is against the official ruleset (which I might have missed, I couldn't find any recommendations in the docu about what should be in recipe config and what not). 

Is there any drawback from having a default config (besides compilation time in dev env)?